packages/fluentui/react-northstar/test/specs/commonTests/isConformant.tsx

```diff
@@ -14,17 +15,19 @@ import { act } from 'react-dom/test-utils';
 import { isExportedAtTopLevel } from './isExportedAtTopLevel';
```

Comment:
Just realized, I think you can also remove the call to this helper (and delete its file)